openex-api/src/main/java/io/openex/injects/email/service/ImapService.java

```diff
@@ -105,10 +105,20 @@ private void initStore(Environment env) throws Exception {
         int port = env.getProperty("openex.mail.imap.port", Integer.class, 995);
         String username = env.getProperty("openex.mail.imap.username");
         String password = env.getProperty("openex.mail.imap.password");
+        String sentFolder = env.getProperty("openex.mail.imap.sent");
```

Comment:
You should be able to use the property defined in line 64 no ?

    @Value("${openex.mail.imap.sent}")
    private String sentFolder;

Reply:
You can't because you are in the constructor method, my mistake
You can let it like that or use the @PostConstructor annotation to have the injected properties

Lifecycle : constructor call, inject properties, post constructor method call